src/utils/getAllProperties.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable guard-for-in */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable no-underscore-dangle */
```

Comment:
I know you're doing some zod digging here to make a great utility, but let's stick to line or block eslint exceptions when necessary instead of the whole file. It makes it easier to see and review why rules are being ignored when the exceptions are tied to the code, and easier to maybe go back and fix.